examples_test.go

```diff
@@ -1,48 +1,45 @@
 package app
 
 import (
-	"fmt"
 	"os"
+	"testing"
 
 	"github.com/docker/app/loader"
 	"github.com/docker/app/render"
+	"github.com/docker/app/types"
 	yaml "gopkg.in/yaml.v2"
+	"gotest.tools/assert"
 )
 
-func Example() {
-	// Load the file (single-file format, there is multiple format)
-	f, err := os.Open("./examples/hello-world/hello-world.dockerapp")
-	if err != nil {
-		panic("cannot read application")
-	}
-	defer f.Close()
-	app, err := loader.LoadFromSingleFile("myApp", f)
-	if err != nil {
-		panic("cannot load application: " + err.Error())
-	}
-	// Render the app to a composefile format, using some user provided parameters
-	c, err := render.Render(app, map[string]string{
-		"text": "hello examples!",
-	}, nil)
-	if err != nil {
-		panic("cannot render application")
-	}
-	// Marshal it to yaml (to display it)
-	s, err := yaml.Marshal(c)
-	if err != nil {
-		panic("cannot marshall the composefile in yaml")
+var examplePaths = map[string]types.AppSourceKind{
```

Comment:
This `Example` function is not what you think it is. There are already e2e tests checking that examples are still up-to-date: https://github.com/docker/app/blob/master/e2e/example_test.go#L13
An Example function is like a special test function, but only checking the stdout with the comment `// Output...`.
It also appears in the godoc.
I think you can safely trash this whole file 👍